modules/rpc/yarpc.go

```diff
@@ -105,78 +176,105 @@ func newYarpcModule(
 		}
 	}
 
-	err := module.Host().Config().Get(fmt.Sprintf("modules.%s", module.Name())).PopulateStruct(cfg)
-	// found values, update module
-	module.config = *cfg
+	err = module.Host().Config().Scope("modules").Get(module.Name()).PopulateStruct(&module.config)
```

Comment:
I know you're going for readability, but block-scoping `errs` should almost always be preferred. could we instead break up this crazy accessor chain a bit for clarity?

```go
mod := module.Host().Config().Scope("modules").Get(module.Name())
if err := mod.PopulateStruct(&module.conig); err != nil {
  return module, err
}
```

also what exactly is this code doing? i know it's loading some config, but the term "module" and "config" show up so many times in a single line it's really hard to tell (to me) exactly what's going on except that we're definitely fetching some configuration for a module